libexec/rtld-cheri-elf/Makefile

```diff
@@ -33,7 +33,7 @@ CFLAGS+=-DNO_LD_DEBUG
 # This could be optimized to use different stubs for per-file stubs but for
 # now making it opt-in is easier.
 # TODO: add a make options for this (e.g. MK_BENCHMARK_OPTIONS)?
-.ifdef RTLD_PER_FUNCTION_CAPTABLE_SUPPORT
+.if defined(RTLD_PER_FUNCTION_CAPTABLE_SUPPORT) && ${TARGET} == "mips"
```

Comment:
I don't like this, it should be an error to request something that's not supported like this... but I guess there's precedent with all the BROKEN_OPTIONS stuff.

Reply:
Should probably not be in this PR, I just noticed it because my config file defines it all "cheribsd" targets.

I can also make it an error for !MIPS, but ignoring it seems simpler (and it's unlikely that anyone except me ever defined that option).